docs/dev/event_time.md

```diff
@@ -146,7 +146,7 @@ to use timestamp assignment and watermark generation in the Flink DataStream API
 *Note: Flink implements many techniques from the Dataflow Model. For a good introduction to event time and watermarks, have a look at the articles below.*
 
   - [Streaming 101](https://www.oreilly.com/ideas/the-world-beyond-batch-streaming-101) by Tyler Akidau
-  - The [Dataflow Model paper](https://static.googleusercontent.com/media/research.google.com/en/pubs/archive/43864.pdf)
```

Comment:
Looks like this should have been `https://research.google.com/pubs/archive/43864.pdf` before the redirect (linked from `https://research.google.com/pubs/pub43864.html`)